docs/source/gbd2017_models/risk_attributable_causes/diabetes_mellitus/index.rst

```diff
@@ -209,6 +242,14 @@ State and Transition Data Tables
      - YLD appendix
      - Disability weight of sequela with id {id}
      - 
+   * - remission_modelable_entity_id_2005
+     - epi
+     - remission of overall Diabetes Mellitus from epi database
+     - 
+   * - incidence_c587
+     - dismod
+     - incidence of overall Diabetes Mellitus
+     - 
 
 Validation Criteria
 -------------------
```

Comment:
Prevalence will be validate against GBD, as will morbidity and mortality.  Given the assumptions described above, we prioritize validation of prevalence.